xiblint/rules/accessibility_format.py

```diff
@@ -7,12 +7,15 @@
     view_accessibility_identifier,
 )
 
+if False:  # if TYPE_CHECKING:
```

Comment:
How does this work? Should there be a follow up to remove `if False`? Or is mypy reading the comment? I don't see anything about commenting it out here: https://mypy.readthedocs.io/en/latest/common_issues.html

Reply:
It is equivalent to https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

however, `typing.TYPE_CHECKING` is not available in python3.5 and I wasn't sure what versions of python `xiblint` supports

`mypy` (and other static type checkers) follow that import, but at runtime it doesn't occur.

I wasn't sure if the import was cyclical or I wouldn't have guarded it by `if False:`, if it's not a cycle the `if False` can be removed entirely

Reply:
Ah so, mypy evaluates all `import`s in a file unconditionally?

Reply:
not all, but those that equate to `if False:` or `if typing.TYPE_CHECKING:` or `if sys.version (comparison) (compatible version):` and  a few other cases like that

Reply:
thanks for mypy education